docs/src/Guides/10 Events.md

```diff
@@ -1,79 +1,138 @@
 
-=== ":one: Decorators"
-    ```python
-    from interactions import listen
-    from interactions.api.events import ChannelCreate
+## Event Listening
 
-    @listen()
-    async def on_channel_create(event: ChannelCreate):
-        # this event is called when a channel is created in a guild where the bot is
+After your intents have been properly configured, you can start to listen to events. Say, if you wanted to listen to channels being created in a guild the bot can see, then all you would have to do is this:
 
-        print(f"Channel created with name: {event.channel.name}")
-    ```
+```python
+from interactions import listen
+from interactions.api.events import ChannelCreate
 
-    You can also use `@listen` with any function names:
+@listen(ChannelCreate)
+async def channel_create_handler(event: ChannelCreate):
+    print(f"Channel created with name: {event.channel.name}")
+```
 
+As you can see, the `listen` statement marks a function to receieve (or, well, listen to) a specific event - we specify which event to receive by passing in the *event object*, which an object that contains all information about an event. Whenever that events happens in Discord, it triggers our function to run, passing the event object into it. Here, we get the channel that the event contains and send out its name to the terminal.
+
+???+ note "Difference from other Python Discord libraries"
+    If you come from some other Python Discord libraries, or even come from older versions of interactions.py, you might have noticed how the above example uses an *event object* - IE a `ChannelCreate` object - instead of passing the associated object with that event - IE a `Channel` (or similar) object - into the function. This is intentional - by using event objects, we have greater control of what information we can give to you.
+
+    For pretty much every event object, the object associated with that event is still there, just as an attribute. Here, the channel is in `event.channel` - you'll usually find the object in other events in a similar format.
+    Update events usually use `event.before` and `event.after` too.
+
+While the above is the recommended format for listening to events (as you can be sure that you specified the right event), there are other methods for specifying what event you're listening to:
+
+???+ warning "Event name format for some methods"
+    You may notice how some of these methods require the event name to be `on_all_in_this_case`. The casing itself is called *snake case* - it uses underscores to indicate either a literal space or a gap between words, and exclusively uses lowercase otherwise. To transform an event object, which is in camel case (more specifically, Pascal case), to snake case, first take a look at the letters that are capital, make them lowercase, and add an underscore before those letters *unless it's the first letter of the name of the object*.
```

Comment:
```suggestion
    You may notice how some of these methods require the event name to be `all_in_this_case`. The casing itself is called *snake case* - it uses underscores to indicate either a literal space or a gap between words, and exclusively uses lowercase otherwise. To transform an event object, which is in camel case (more specifically, Pascal case), to snake case, first take a look at the letters that are capital, make them lowercase, and add an underscore before those letters *unless it's the first letter of the name of the object*.
```

This is confusing for people to read. Removed the `on_`